pkg/apis/networking/register.go

```diff
@@ -73,6 +73,15 @@ const (
 	// value a different reconciliation logic may be used (for examples,
 	// Cert-Manager-based Certificate will reconcile into a Cert-Manager Certificate).
 	CertificateClassAnnotationKey = GroupName + "/certificate.class"
+
+	// WildcardCertLabelKey is the label key attached to a namespace to indicate that
+	// a wildcard certificate should be created for it.
+	WildcardCertLabelKey = GroupName + "/wildcardCert"
+
+	// OwnerNamespaceLabelKey is the label attached to Certificates to
+	// differentiate Certificates created by the Route controller from Certificates
+	// created by the Namespace controller.
+	OwnerNamespaceLabelKey = "networking.knative.dev/ownerNamespace"
```

Comment:
We should let OwnerReferences do this, they have a Kind.

Reply:
I see. We can use OwnerReferences here. Originally we considered that OwnerReferences is a kind of redundant from GC's perspective.